source/AI.cpp

```diff
@@ -1539,6 +1543,20 @@ shared_ptr<Ship> AI::FindTarget(const Ship &ship) const
 				&& foe->IsDisabled() && (!canPlunder || Has(ship, foe->shared_from_this(), ShipEvent::BOARD)))
 			continue;
 
+		foe->UpdateTargeterStrength();
+		double targeterStrength = foe->GetTargeterStrength();
+		int targeterCount = foe->GetShipsTargetingThis().size();
+
+		// The next two checks only apply if more than two ships are attacking the foe, as well as
+		// if it has not already been selected as a target recently.
+		// Deprioritize this if more than a quarter of your allies' strength is already attacking.
+		if(targeterStrength >= 0.25 * alliedStrength && targeterCount > 2 && !preferredTarget)
+			range += 500;
+
+		// Deprioritize this if more than twice its strength are already attacking it.
```

Comment:
I wound up here looking at the code because I found this language confusing

```suggestion
		// Deprioritize this if its attackers outmatch it two-to-one.
```

Reply:
Case if you accept my 3-to-one suggestion.

```suggestion
		// Deprioritize this if its attackers outmatch it three-to-one.
```

Reply:
The reason why it doesn’t say that is because that isn’t correct. Stronger (read: more expensive) ships count more towards “three ships” (in your case) than weaker ones.

Reply:
I don't follow what you mean by "counting towards three ships".  The three ships check looks separate from the strength check to me?

The way the lines below *look* to me, is that "if this ship's attackers total strength is more than twice this ship's strength, and there are already more than two ships attacking it, consider this target as if it were 500 pixels further away than it actually is"

```suggestion
		// Deprioritize this if the ships attacking it are more than twice as strong.
```

Reply:
Yeah, that’s what I was getting at. Thank you.

Reply:
The language is confusing.  I may understand it now after this back-and-forth, but those coming in the future may not be here to do so.   If not my suggestion, then it ought to be reworded somehow.